src/it/resources-noncompilable/org/checkstyle/suppressionxpathfilter/packageannotation/SuppressionXpathRegressionPackageAnnotationTwo.java

```diff
@@ -0,0 +1,6 @@
+//non-compiled with javac: non-compilable annotation, should be in file package-info.java
+@Deprecated
+@SuppressWarnings
```

Comment:
please create one more `package-info.java` and do validation on it. 

all should be compilable. you can create as much packages as we need in Input folder.

Reply:
Understood.